playbooks/roles/unbound/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install unbound and resolvconf
+  apt:
+    name={{ item }}
+    state=installed
+  with_items:
+    - unbound
+    - resolvconf
+
+- name: Copy the configuration for caching and forwarding
+  template:
+    src: cache-and-forward.conf.j2
+    dest: "{{ unbound_config_dir }}/cache-and-forward.conf"
+
+- name: restart unbound
```

Comment:
This should be a handler and only triggered by notification if the template changes.

Reply:
@UmanShahzad, that is correct. Will fix this.